src/app/app.module.ts

```diff
@@ -15,6 +14,7 @@ import { Broadcaster, Logger } from 'ngx-base';
 import { DropdownModule } from 'ngx-dropdown';
 import { AuthenticationService, UserService, HttpService } from 'ngx-login-client';
 import { WidgetsModule } from 'ngx-widgets';
+import { MomentModule } from 'angular2-moment';
```

Comment:
We are already using the "moment" library is the other modules. If they do the same thing we should use just one to keep from downloading 2 libraries that do the same thing.